migrate_pim/migrate_to_saas.rst

```diff
@@ -0,0 +1,139 @@
++--------------------------------+---------------------------------------------------------------------------------------+
+
+Disable custom code
+~~~~~~~~~~~~~~~~~~~
+
+Serenity does not support custom code within the PIM. In Serenity, all custom features use API connections and apps from the `App Store <https://apps.akeneo.com>`_ 
+(including `custom apps <https://api.akeneo.com/apps/create-custom-app.html>`_).
+
+**If you have custom code bundles, please** `remove them <https://docs.akeneo.com/master/maintain_pim/first_aid_kit/index.html#step-10-did-you-customize-your-pim>`_ 
+**before you begin the migration process.**
+
+If you cannot find a suitable replacement for your custom code, please contact your Customer Success Manager — they may be able to suggest changes
+or offer solutions that are compatibile with Serenity.
+
+.. warning::
+
+    We cannot validate your instance's migration to Serenity without all of the following information.
+    If we request any changes, we may ask you to run some commands again to check the updated status of your PIM.
+
+Check your database schema
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Please send us the results of the following shell commands so that we can ensure your database schema is up-up-to-date and compatible with a migration.
+
+.. code:: bash
+
+    $ cd /home/akeneo/pim
+    $ bin/console pimee:database:inspect -f --env=dev
+    $ composer require jfcherng/php-diff
+    $ bin/console pimee:database:diff --env=dev
+    $ bin/console doctrine:migrations:status
+
+.. warning::
+
+    If the results of ``bin/console doctrine:migrations:status`` show a non-zero number of "New" migrations, your
+    database is not up to date. Please run ``bin/console doctrine:migrations:migrate``, run the ``status`` command again,
+    and send us the output of all of these commands so that we can see any changes.
+
+Check your filesystem
+~~~~~~~~~~~~~~~~~~~~~
+
+We also require the output of the following commands, so that we can check to make sure your filesystem adapter is set up correctly for the migration to Serenity.
+
+.. code:: bash
+
+    $ cd /home/akeneo/pim
+    $ bin/console debug:config OneupFlysystemBundle
+
+Schedule a timeslot for migration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Akeneo Support team **must validate** the output of the above commands before we can schedule any migration.
+
+Once we have received all of this information and validated it, we can schedule the Flexibility to Serenity migration operation with our Cloud Engineering team. 
+Their hours are **Monday to Friday, 9:00 am to 6:00 pm CET**. Please provide at least 48 hours notice between your request and the actual migration time slot.
+This gives our team time to prepare and to ensure that your migration runs smoothly.
+
+We usually estimate 2 hours for each migration, although the duration of individual migration operations can vary, especially if your catalog is very large or complex.
```

Comment:
it's a very optimistic view.
according to past migrations, for most of our customers, it took between 2 hours to 4 hours.

Reply:
Good point. I'll adjust it. 👍🏻

Reply:
```suggestion
For most instances, migrations will take between 2 and 4 hours. However, the duration of individual migration operations can vary, especially if your catalog is very large or complex.

If you have concerns about the timing of a migration, please let us know in the migration Support ticket and we will work with you to find the best solution.
```